### PR TITLE
imzML partitioner

### PR DESCRIPTION
**Pull request by abourramouss**

### CODE REVIEW

File: dataplug/metabolomics/imzML.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    
+def pt_strat(cloud_object: IMZML):
+    
+    body_imzml = cloud_object.s3.get_object(Bucket=cloud_object._obj_path.bucket, Key=cloud_object._obj_path.key)['Body']
+    tmp = tempfile.NamedTemporaryFile()
+    with open(tmp.name, 'wb') as f:
+        f.write(body_imzml.read())
+    parser = ImzMLParser(tmp.name, ibd_file=None)
+    position = parser.coordinates
+    slices = []
```
Comment (aitorarjona) [inline dataplug/metabolomics/imzML.py:153]: convert to list comprehension

File: dataplug/metabolomics/imzML.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        position = parser.coordinates.index(self.coordinate)
+
+        mzOffset = parser.mzOffsets[position]
+        mzSize = (parser.sizeDict.get(parser.mzPrecision) * parser.mzLengths[position]) - 1
+
+        # Download our specific data.
+        header_request = self.s3.get_object(Bucket=self.obj_path.bucket, Key=key_ibd,
+                                                 Range='bytes={}-{}'.format(mzOffset,
+                                                                            mzOffset + mzSize))
+        mzArray = []
```
Comment (aitorarjona) [inline dataplug/metabolomics/imzML.py:108]: convert to list comprehension

**Review by aitorarjona — CHANGES_REQUESTED** [review]
remove sample files or find some smaller files ~10KB max

File: dataplug/metabolomics/imzML.py (hunk 2 above)
Comment (locotron) [reply dataplug/metabolomics/imzML.py:108]: If converted to list comprehension it will become List[np.buffer] instead of np.array.

**Pull request closed without merging**